README.rst

```diff
@@ -153,12 +153,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Magen, Yohai and
```

Comment:
A bit late but just wanted to say that I'm perfectly happy not being first since @seisman @weiji14 @meghanrjones and others deserve it more than I do 👍🏽 Creating something is easy, keeping it going and growing like this is where the real work is 🙂

Reply:
Thanks Leo for chiming in! I would argue that you still deserve the first spot since the original GMT C to Python interface (via ctypes) is so foundational (some of that 4+ year old code is still more or less intact at https://github.com/GenericMappingTools/pygmt/blame/v0.5.0/pygmt/clib/session.py and working!!!), and this is what makes this 'PyGMT' package so good compared to the other gmtpy/pygmt wrappers people have tried to come up with previously.

But again, I don't want to get into an author ordering debate, I'm just as happy being on third place, last place, alphabetically ordered or have a random number generator sort things out. We've discussed the AUTHORSHIP.md policy before at https://github.com/GenericMappingTools/pygmt/pull/726#issuecomment-776076058, and could edit it again if needed, especially when the time comes closer to getting a software publication out.